src/api/journeys.ts

```diff
@@ -45,7 +47,14 @@ export const journeyInstanceResource = (orgId: string, instanceId: string) => {
 
   return {
     prefetch: createPrefetch<ZetkinJourneyInstance>(key, url),
+    useAddAssignee: createUseMutationPut({ key, url: `${url}/assignees` }),
+    useAddMember: createUseMutationPut({ key, url: `${url}/subjects` }),
```

Comment:
I think we should call this `useAddSubject`, to be consistent with what it's called in the API.